### DISCUSSION TITLE
4326 tile services

### DISCUSSION THREAD

**Discussion by xyzhuo** [maplibre/maplibre-gl-js#4656] [General]
Please ask, does Maplibre support loading 4326 tile services? Or how can I make it support? Such as 4326 wmts services.

**Comment by HarelM** [comment]
wtms is supported, I advise to take a look at the examples and style spec docs.

**Reply by xyzhuo** [reply]
I can't find an example that supports loading 4326 slicing schemes. Can you give me a link?

**Reply by HarelM** [reply]
Check scheme parameter:
https://maplibre.org/maplibre-style-spec/sources/#scheme

**Reply by xyzhuo** [reply]
The scheme parameter can only set the reverse of the y-axis, that is, the xyz or tms standard. This parameter does not specify the tile splitting method of 4326 (WGS84). Usually, the tile splitting method has 0 levels and 2 tiles. You can see this example of openlayers: https://openlayers.vip/examples/vector-tiles-4326.html, it sets a coordinate system parameter projection: 'EPSG: 4326'.
